docs/sources/configure-server/anonymous-usage-statistics-reporting.md

```diff
@@ -0,0 +1,60 @@
+---
+
+description: Learn about Grafana Pyroscope anonymous usage statistics reporting
+menuTitle: Anonymous usage statistics reporting
+title: About Grafana Pyroscope anonymous usage statistics reporting
+weight: 30
+---
+
+# About Grafana Pyroscope anonymous usage statistics reporting
+
+Grafana Pyroscope includes a system that optionally and anonymously reports non-sensitive, non-personally identifiable information about the running Pyroscope cluster to a remote statistics server.
+Pyroscope maintainers use this anonymous information to learn more about how the open source community runs Pyroscope and what the Pyroscope team should focus on when working on the next features and documentation improvements.
+
+The anonymous usage statistics reporting is **enabled by default**.
+You can opt-out setting the CLI flag `-usage-stats.enabled=false` or its respective YAML configuration option.
+
+## The statistics server
+
+When usage statistics reporting is enabled, information is collected by a server that Grafana Labs runs. Statistics are collected at `https://stats.grafana.org`.
+
+## Which information is collected
+
+When the usage statistics reporting is enabled, Grafana Pyroscope collects the following information:
+
+- Information about the **Pyroscope cluster and version**:
```

Comment:
In Tempo, we've clarified this information to show which configurations values are used: receivers, frontend concurrency and version, storage cache, backend, WAL and block encodings, ring replication factor, KVstore, feature toggles enabled. It's a smaller list, but might be easier for people to sort through. 

Doc in question: https://grafana.com/docs/tempo/latest/configuration/#usage-report

Reply:
Given our other conversation, I'll update the Tempo docs to be more transparent in this detailed section.